ntcore/src/main/native/cpp/Value.cpp

```diff
@@ -317,6 +317,10 @@ bool nt::operator==(const Value& lhs, const Value& rhs) {
       if (lhs.m_val.data.arr_boolean.size != rhs.m_val.data.arr_boolean.size) {
         return false;
       }
+      if (lhs.m_val.data.arr_boolean.size == 0 ||
+          rhs.m_val.data.arr_boolean.size == 0) {
```

Comment:
Since we changed this to check size, we actually don't need the || at all.  We already check the two sides are equal in size, so we only need to check that one of the sides is 0.